uktrade/data-hub-frontend
# Add Lead ITA confirmation page

## Description of change

Added the Add Lead ITA page.

## Test instructions

1. Go to `/companies/<id>/advisers` and hit the _Add myself as Lead ITA_  button
2. It should take you to the new _Add Lead ITA _ page at `/companies/<id>/advisers/add`
3. Both the _cancel_ and _Add myself as Lead ITA_ buttons should take you back to `/companies/<id>/advisers`
4. `/companies/<id>/advisers/add` should return `404` if the feature flag `lead_advisers` is not enabled.
5. `/companies/<id>/advisers/add`shoudl return `403` if the user doesn't have the `company.change_regional_account_manager` permission.
 
## Screenshots

### Before

There page didn't exist before.

### After 

<img width="697" alt="Screen Shot 2019-11-15 at 3 10 47 PM" src="https://user-images.githubusercontent.com/2333157/68953825-a1516900-07ba-11ea-8ff4-15d330b17420.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
